app/experimenter/static/js/components/DesignForm.js

```diff
@@ -106,13 +106,15 @@ class DesignForm extends React.PureComponent {
         </Container>
       );
     }
-
     // Select the appropriate form component to use based on experiment type
     let Form;
     switch (this.state.data.get("type")) {
       case "pref":
         Form = PrefForm;
         break;
+      case "multi-pref":
```

Comment:
Is multi-pref a separate experiment type?

Reply:
Note: we chatted and talked about passing this in as a prop to DesignForm through the template the way we do for `is_branched_addon`